test/Microsoft.Extensions.Hosting.Tests/HostTests.cs

```diff
@@ -505,6 +505,67 @@ public void HostCanBeStarted()
         }
 
         [Fact]
+        public async Task HostStopAsyncUsesDefaultTimeoutIfGivenTokenDoesNotFire()
```

Comment:
Nit: Shorten the `Task.Delay(TimeSpan.FromSeconds(8))` in `HostStopAsyncCanBeCancelledEarly` to be less than 5 seconds.